roles/openshift_docker/tasks/main.yml

```diff
@@ -9,7 +9,8 @@
   when: openshift.common.is_containerized is defined and openshift.common.is_containerized | bool and openshift_image_tag is not defined
 
 - set_fact:
-    l_image_tag: "{{ cli_image_version.stdout_lines[0].split(' ')[1].split('-')[0] }}"
+    l_image_tag: "{{ cli_image_version.stdout_lines[0].split(' ')[1].split('-')[0:2] | join('-') if openshift.common.deployment_type == 'origin' else
```

Comment:
In general or after my change? I think we only need concern ourselves with Origin here, OSE variants are so coupled to RPM build process that I cannot foresee us ever relying on a second version component mapping to image versions.

So, for origin I looked at 'latest', 'v1.1.6', and 'v1.2.0-rc1' and this should work for them all.